setup.py

```diff
@@ -88,6 +88,11 @@ def pybindings(cls) -> bool:
     def xnnpack(cls) -> bool:
         return cls._is_env_enabled("EXECUTORCH_BUILD_XNNPACK", default=False)
 
+    @classmethod
+    @property
+    def llama_custom_ops(cls) -> bool:
```

Comment:
Would be good to remove `llama_` here since that name isn't part of the env vars.